server/functions.lua

```diff
@@ -358,7 +358,7 @@ function ToggleOptin(source)
     if not license or not IsPlayerAceAllowed(source --[[@as string]], 'admin') then return end
     local player = GetPlayer(source)
     player.PlayerData.optin = not player.PlayerData.optin
-    player.Functions.SetPlayerData('optin', player.PlayerData.optin)
+    SetPlayerData(player.PlayerData.source, 'optin', player.PlayerData.optin, nil, true)
```

Comment:
Never really a fan of passing in nil/null values into methods, rife for ungraceful exiting.

Reply:
Couldn't find another way to keep backwards compatibility

Reply:
I understand, sucks dick tho